server.js

```diff
@@ -0,0 +1,7 @@
+const http = require('http');
+// var url = require('url');
+const app = require('./lib/app');
+
+http.createServer(app)
+  .listen(8888);
```

Comment:
```suggestion
  .listen(PORT, () => console.log(`Server running on port ${PORT}`));
```
Sometimes helps to have some visual feedback when starting up your server (or port number, etc.)